<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PMP-Farmácias</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../logo_pmp.ico</normaloff>../../../../../logo_pmp.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>421</width>
     <height>331</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="userInput">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color:#fff;
background-color: rgba(0, 0, 0, 150);
border-radius:3.5px;
padding:8px;</string>
    </property>
    <property name="placeholderText">
     <string>Usuário:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwdInput">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color:#fff;
background-color: rgba(0, 0, 0, 150);
border-radius:3.5px;
padding:8px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Senha:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:3.5px;
	background-color:green;
	color:white;
}
QPushButton:hover{
	
	background-color:rgb(0,200,0);
	color:black;
}</string>
    </property>
    <property name="text">
     <string>ENTRAR</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>271</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../Downloads/logo_pmp.png</pixmap>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>-11</x>
     <y>-1</y>
     <width>651</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/wallpaper_pmp.jpg);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources>
  <include location="../../../py_images/images.qrc"/>
 </resources>
 <connections/>
</ui>
